### ISSUE TITLE
html align tag: gdistance scheduled for CRAN archival

### ISSUE THREAD

**Issue by jsta** [AgrDataSci/gdistance#13]
As the maintainer of the `ipdw` package, I received a message from CRAN that gdistance is scheduled for archival because of the following CRAN check NOTE:

```
checking HTML version of manual ... [4s] NOTE
Found the following HTML validation problems:
gdistance.html:23:4: Warning: <img> attribute "align" not allowed for HTML5
```

**Comment by kauedesousa** [comment]
There are other tiny silly issues that I need to fix. I sent a version to CRAN two weeks ago but it returned to me and I was without internet connection to submit a new version. I'll work on that tomorrow.

**Comment by jsta** [comment]
Seems hardly true that they've had "no reply from the maintainer [of gdistance]" then 😅 . Thanks for your work.

**Comment by kauedesousa** [comment]
I was in communication with them. But I had very hectic weeks with several travels to different countries, added to well deserved break this week, which will be interrupted due to this final call from CRAN

**Comment by kauedesousa** [comment]
These are the other issues identified 

Flavor: r-devel-linux-x86_64-debian-gcc, r-devel-windows-x86_64
Check: CRAN incoming feasibility, Result: NOTE
  Maintainer: 'KauÃª de Sousa <desousa.kaue@gmail.com>'
  
  New maintainer:
    KauÃª de Sousa <desousa.kaue@gmail.com>
  Old maintainer(s):
    KauÃª de Sousa <kaue.desousa@inn.no>

Flavor: r-devel-linux-x86_64-debian-gcc, r-devel-windows-x86_64
Check: R code for possible problems, Result: NOTE
  Found if() conditions comparing class() to string:
  File 'gdistance/R/commuteDistance.R': if (class(transitionMatrix(x)) != "dsCMatrix") ...
  File 'gdistance/R/geoCorrection.R': if (class(transitionMatrix(x)) == "dsCMatrix") ...
  File 'gdistance/R/pathInc.R': if (class(weight) == "numeric") ...
  File 'gdistance/R/pathInc.R': if (class(weight) == "TransitionLayer") ...
  File 'gdistance/R/pathInc.R': if (class(weight) == "TransitionStack") ...
  File 'gdistance/R/transition.R': if (class(transitionFunction) == "character") ...
  Use inherits() (or maybe is()) instead.

**Comment by kauedesousa** [comment]
The issue is fixed with the new version 1.6 just published on CRAN
